src/App.js

```diff
@@ -0,0 +1,20 @@
+/* App.js */
+
+import Graph from './components/Graph';
+var React = require('react');
+var Component = React.Component;
+ 
+export default class App extends Component {
+  render() {
+    return (
+      <div>
+      <h1>ShipNoise.net</h1>
+      <h3>Characterizing ship noise in orca habitats</h3>
+      
+      <p>Ship noise dominates the acoustic habitat of the Southern Resident Orca Whales within the Salish Sea.</p>
```

Comment:
This should probably use the more (scientifically) conventional "Southern Resident Killer Whales" and leave "orcas" for the next line. That will be good for SEO and the two lines together will "speak" to biological experts and citizen scientists, alike.